app/views/organisations/_show_organisation.html.erb

```diff
@@ -130,6 +130,21 @@
   <% end %>
 <% end %>
 
+<% if @show.high_profile_groups[:items] %>
+  <div class="grid-row organisations__section">
+    <div class="column-two-thirds">
+      <div class="organisations__brand-border-top brand--<%= @organisation.brand %> brand__border-color">
+        <%= render "govuk_publishing_components/components/heading", {
+          text: @show.high_profile_groups[:title],
+          padding: true
```

Comment:
Should we add some margin to the bottom of this heading? (i.e. with the `margin:` option). Looks a little squashed. I've been using margin 2 on most of the other headings.